server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewAssignmentServicesImpl.java

```diff
@@ -66,8 +71,26 @@ public Set<ReviewAssignment> findAllByReviewPeriodIdAndReviewerId(UUID reviewPer
         return reviewAssignments;
     }
 
+    @Override
+    public ReviewAssignment update(ReviewAssignment reviewAssignment) {
+        LOG.warn(String.format("Updating entity %s", reviewAssignment));
```

Comment:
Is `warn` the appropriate method here? That seems to indicate something is wrong.